docs/getting-started/backends-and-brokers/sqs.rst

```diff
@@ -150,6 +150,41 @@ setting::
         }
     }
 
+Back-off policy
+------------------------
+Back-off policy is using SQS visibility timeout mechanism altering time diff between task retries.
+
+Configuring the queues and backoff policy::
+
+    broker_transport_options = {
```

Comment:
some more detail about where to configure this would be good for the end users

Reply:
Added both here and https://github.com/celery/kombu/pull/1301